api/app/Traits/Generator/Filterable.php

```diff
@@ -11,6 +11,18 @@ trait Filterable
 
     protected ?array $ids = null;
 
+    /** If true, stores applied filters to debugging */
+    protected bool $debug = false;
+
+    public array $calledFilters = [];
+
+    public function debug(bool $value)
+    {
+        $this->debug = $value;
+
+        return $this;
+    }
```

Comment:
Is this something we intend to keep, or was this meant to be deleted after a problem was found?

Reply:
Unfortunately we need to keep it so we can use it in the test :cry: